### PR TITLE
[DFFS-10] The Software Developer conditionals

### PR DESCRIPTION
**Pull request by RafiatAmin**
@adaugochi @Ireti203 kindly review my PR

### CODE REVIEW

**Comment by Ireti203** [comment]
Attach the screenshot of your console

File: submissions/rafiatAminu/JavaScript/cond.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
+let a = 0;
+let b= 7;
+let c= 123;
+let d= (c-b);
+
+for(a=0; a<d;) {
```
Comment (adaugochi) [inline submissions/rafiatAminu/JavaScript/cond.js:6]: The `for()` take 3 arguments, two given. Also, kindly format this block of code to make it readable

**Review by adaugochi — CHANGES_REQUESTED** [review]

**Comment by RafiatAmin** [comment]
Screenshot 2021-03-30 141232

@adaugochi @Ireti203 Kindly view

File: submissions/rafiatAminu/JavaScript/cond.js [hunk 2]
```diff
@@ -0,0 +1,25 @@
+let a = 0;
```
Comment (adaugochi) [inline submissions/rafiatAminu/JavaScript/cond.js:1]: what's this variable doing? it doesn't have any usage

File: submissions/rafiatAminu/JavaScript/cond.js [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+let b= 7;
+let c= 123;
+let d= (c-b);
```
Comment (adaugochi) [inline submissions/rafiatAminu/JavaScript/cond.js:4]: variable names should be descriptive enough. things like `a, b, c d` etc as a variable name is not proper

File: submissions/rafiatAminu/JavaScript/cond.js (hunk 2 above)
Comment (RafiatAmin) [reply submissions/rafiatAminu/JavaScript/cond.js]: I have taken it out

File: submissions/rafiatAminu/JavaScript/cond.js (hunk 3 above)
Comment (RafiatAmin) [reply submissions/rafiatAminu/JavaScript/cond.js]: I have worked on that as well. Kindly review @adaugochi Thanks

**Review by Ireti203 — APPROVED** [review]

**Review by RafiatAmin — COMMENTED** [review]
Thank you, but it’s not merged yet

**Review by RafiatAmin — COMMENTED** [review]
Thanks

**Pull request merged**